app/src/main/java/ch/epfl/sdp/MainActivity.kt

```diff
@@ -0,0 +1,26 @@
+package ch.epfl.sdp
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import ch.epfl.sdp.databinding.ActivityMainBinding
+import ch.epfl.sdp.lobby.LobbyActivity
+
+class MainActivity : AppCompatActivity() {
+    private var _binding: ActivityMainBinding? = null
+    private val binding get() = _binding!!
```

Comment:
Prefer using a lateinit in the context of activities, it prevent from having nullable variables